conf/fail2ban/jail.conf

```diff
@@ -56,14 +98,17 @@ ignoreip = 127.0.0.1/8
 ignorecommand =
 
 # "bantime" is the number of seconds that a host is banned.
-bantime  = 600
+bantime  = 10m
 
 # A host is banned if it has generated "maxretry" during the last "findtime"
 # seconds.
-findtime  = 600
+findtime  = 10m
 
 # "maxretry" is the number of failures before a host get banned.
-maxretry = 10
+maxretry = 5
```

Comment:
I think `10` was a feature maybe, like 5 is too harsh, too many people end up being banned and complain about it on the support ...